kairon/shared/channels/mail/data_objects.py

```
@@ -0,0 +1,127 @@
+                    subjects=subjects,
+                    classification_prompt=classification_prompt,
+                    reply_template=reply_template,
+                    bot=bot,
+                    timestamp=time.time(),
+                    status=True,
+                    user=user
+                )
+                mail_config.save()
+
+        except Exception as e:
+            raise AppException(str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)